### PR TITLE
[NFC][SYCL] Remove explict "cl::" namespace references

### PR DESCRIPTION
**Pull request by aelovikov-intel**
Part of eliminating "cl" namespace according to the SYCL 2020
specification.

### CODE REVIEW

**Review by asudarsa — DISMISSED** [review]
LGTM. Thanks

**Comment by aelovikov-intel** [comment]
Ping everybody.

File: SYCL/Assert/assert_in_kernels.cpp [hunk 1]
```diff
@@ -16,7 +16,7 @@
 // CHECK-NOT:  test aborts earlier, one shouldn't see this message
 // CHECK-NOT:  The test ended.
 //
-// CHECK-ACC-NOT: {{.*}}assert_in_kernels.hpp:26: void kernelFunc2(int *, int): {{.*}} [{{[0,2]}},0,0], {{.*}} [0,0,0]
+// CHECK-ACC-NOT: {{.*}}assert_in_kernels.hpp:25: void kernelFunc2(int *, int): {{.*}} [{{[0,2]}},0,0], {{.*}} [0,0,0]
```
Comment (asudarsa) [inline SYCL/Assert/assert_in_kernels.cpp:19]: Do we really  need to check the line number here? Can we not have {{.*}}? Same comment for couple of changes below as well. Thanks

Comment (aelovikov-intel) [reply SYCL/Assert/assert_in_kernels.cpp:19]: Well, I think we want to verify that it's correct.

**Review by asudarsa — APPROVED** [review]
LGTM. Thanks

**Review by againull — APPROVED** [review]

**Comment by aelovikov-intel** [comment]
@intel/dpcpp-esimd-reviewers , ping.

**Review by v-klochkov — APPROVED** [review]

**Pull request merged**